service/framework_test.go

```diff
@@ -85,6 +85,15 @@ func Test_Service_NewVersionedResources(t *testing.T) {
 					"servicev2",
 					"endpointsv2",
 				},
+				"2.0.3": []string{
+					"kmskeyv2",
+					"s3bucketv2",
+					"s3objectv3",
+					"cloudformationv2",
```

Comment:
We also need a `cloudformationv3` resource that uses `k8scloudconfigv5`. This is so the code is isolated and there are separate code paths for `2.0.2` and `2.1.0`.